terraform/modules/bastion/user_data.sh

```
@@ -1,57 +1,90 @@
 #!/bin/bash
-yum update -y
+set -euo pipefail
+
+BOOTSTRAP_SCRIPT="/usr/local/bin/bastion-bootstrap.sh"
+LOG_FILE="/var/log/bastion-bootstrap.log"
```

Shellcheck (0.11.0)

[warning] LOG_FILE appears unused. Verify use (or export if used externally).

(SC2034)